packages/nextjs/app/builders/0xE474023791245Ff654167397Ea8262B891Ba207c/page.tsx

```diff
@@ -0,0 +1,58 @@
+import Link from "next/link";
+import { NextPage } from "next";
+import { Address } from "~~/components/scaffold-eth";
+
+const CypherpunkPage: NextPage = () => {
+  return (
+    <main className="min-h-screen bg-black text-gray-200 font-mono px-4 py-10">
+      <div className="max-w-4xl mx-auto space-y-8">
+        <h1 className="text-4xl font-bold text-center">🕶️ neeraj27.eth Portfolio</h1>
+
+        {/* Core Values Card */}
+        <div className="bg-zinc-900 rounded-2xl shadow-md p-6 hover:shadow-lg hover:bg-zinc-800 transition-all">
```

Comment:
You should only have hover effects for interactive elements. This hover effect misleads the user into thinking they can click it to navigate.